dotcom-rendering/src/web/components/App.tsx

```diff
@@ -324,6 +325,27 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 				}
 			});
 
+			onConsentChange((consentState) => {
```

Comment:
Subjective, but I think this could be made more readable through some added whitespace and also inlining and renaming. E.g.

```
onConsentChange((consentState) => {
	const consentUUID = getCookie('consentUUID')
	const consentString = consentState?.tcfv2?.tcString || "undefined";

	const event = {
		component: {
		  componentType: 'CONSENT',
		  products: [],
		  labels:  [consentUUID, consentString],
		},
		action: 'CONSENT',
	}

	const record = getOphanRecordFunction()
	record(event);
});
```

Caveat - `consentString` is slightly different than before in behaviour - you may not want this!